### DISCUSSION TITLE
Please integrate GUI like hlky and expsoe the turbo mode to the GUI.

### DISCUSSION THREAD

**Discussion by mhnoni** [basujindal/stable-diffusion#146] [Ideas]
Unless there is already one but I have no idea how to run it.

**Comment by bitRAKE** [comment]
After you've install the repo, activate the conda environment then do ```pip install gradio``` and then you can run the gradio scripts that have a UI (in your web browser).

**Reply by mhnoni** [reply]
> After you've install the repo, activate the conda environment then do `pip install gradio` and then you can run the gradio scripts that have a UI (in your web browser).

Thanks a lot, I will try it.
